### PR TITLE
doc: pull in admin guide

### PR DESCRIPTION
**Pull request by garlick**
Problem: the flux admin guide contains flux-core version specific warnings and generally is closely tied to flux-core development, but it exists in another repo and tends to be forgotten.

Pull it in to flux-core.  Readthedocs can then provide a flux-core version specific link to the document.

I'll follow this up with a commit to remove it from flux-docs, and we can discuss in that one what might be needed to redirect here, if anything.

### CODE REVIEW

**Comment by garlick** [comment]
Hmm, this popped up in the system builder.     
```
2024-03-01T17:15:28.1938625Z FAIL: t9000-system.t 18 - 0005-exec.t: start a long-running guest job
2024-03-01T17:15:28.1939045Z #	
2024-03-01T17:15:28.1939368Z #	        flux submit -n1 --wait-event=start sleep inf &&
2024-03-01T17:15:28.1939755Z #		jobid=$(flux job last)
2024-03-01T17:15:28.1940006Z #	
2024-03-01T17:15:28.1940322Z ok 19 - 0005-exec.t: flux exec --jobid fails as guest
2024-03-01T17:15:28.1940890Z PASS: t9000-system.t 19 - 0005-exec.t: flux exec --jobid fails as guest
2024-03-01T17:15:28.1941574Z ok 20 - 0005-exec.t: flux exec --jobid fails as instance owner
2024-03-01T17:15:28.1942219Z PASS: t9000-system.t 20 - 0005-exec.t: flux exec --jobid fails as instance owner
2024-03-01T17:15:28.1942888Z ok 21 - 0005-exec.t: flux exec without --jobid works as instance owner
2024-03-01T17:15:28.1943869Z PASS: t9000-system.t 21 - 0005-exec.t: flux exec without --jobid works as instance owner
2024-03-01T17:15:28.1944544Z ok 22 - 0005-exec.t: flux exec without --jobid fails as guest
2024-03-01T17:15:28.1945185Z PASS: t9000-system.t 22 - 0005-exec.t: flux exec without --jobid fails as guest
2024-03-01T17:15:28.1945750Z not ok 23 - 0005-exec.t: cancel long-running job
2024-03-01T17:15:28.1946264Z FAIL: t9000-system.t 23 - 0005-exec.t: cancel long-running job
2024-03-01T17:15:28.1946661Z #	
2024-03-01T17:15:28.1946858Z #		flux cancel $jobid
2024-03-01T17:15:28.1947099Z #	
2024-03-01T17:15:28.1947307Z # failed 2 among 23 test(s)
2024-03-01T17:15:28.1947569Z 1..23
2024-03-01T17:15:28.1947869Z ERROR: t9000-system.t - exited with status 1
```

File: doc/conf.py [hunk 1]
```diff
@@ -55,26 +55,27 @@
     'linux:man1': {
         'text': '%s(1)',
         'url': 'https://linux.die.net/man/1/%s',
+        'url': 'http://man7.org/linux/man-pages/man1/%s.1.html'
```
Comment (grondo) [inline doc/conf.py:58]: Commit message typo: s/hte/the/

**Review by grondo — APPROVED** [review]
Nice! Approved!

**Comment by garlick** [comment]
Fixed the typo and also a broken link to the accounting guide.

**Pull request merged**